examples/geospatial/plot_osmnx.py

```diff
@@ -3,15 +3,47 @@
 OSMnx
 =====
 
-This example shows how to read a shapefile with OSMnx
-and convert it to a NetworkX graph....
+This example shows how to use OSMnx to download and model a street network
+from OpenStreetMap, visualize centrality, and save the graph as a shapefile,
+a GeoPackage, or GraphML.
+
+OSMnx is a Python package to retrieve, model, analyze, and visualize
+OpenStreetMap street networks as NetworkX MultiDiGraph objects. It can also
+retrieve any other spatial data from OSM as geopandas GeoDataFrames. See
+https://osmnx.readthedocs.io/ for OSMnx documentation and usage.
 """
 
-import osmnx as ox
 import networkx as nx
+import osmnx as ox
+
+ox.config(use_cache=True, log_console=True)
+
+# download street network data from OSM and construct a MultiDiGraph model
```

Comment:
Is this comment correct?

Reply:
I believe so.  I didn't look at the implementation, but if I turn off my wireless network it produces the following error
```
In [1]:  import osmnx as ox

In [2]:  G = ox.graph_from_point((37.79, -122.41), dist=750, network_type="drive")
...
ConnectionError: HTTPConnectionPool(host='overpass-api.de', port=80): Max retries exceeded with url: /api/status (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7e6de14c10>: Failed to establish a new connection: [Errno -2] Name or service not known'))
...
```
and if I reenable it I get
```
In [3]:  G = ox.graph_from_point((37.79, -122.41), dist=750, network_type="drive")

In [4]: G
Out[4]: <networkx.classes.multidigraph.MultiDiGraph at 0x7f7e6c8286a0>
```

Reply:
Yes, it is correct.